src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -229,6 +226,28 @@ def _clean_all_data_present_for_publish(self):
                 ),
             )
 
+        from ..utils import has_overlapping_objects
+
+        if has_overlapping_objects(
```

Comment:
This needs to be put here so it does not published overlapping zaaktypes but  `has_overlapping_objects` is called 3 times when you click publish with the admin.

Publish admin mixin is implemented confusingly and I can not think of a better way.

Reply:
Actually #1532  wants a separate publishing view, so this can be fixed there.